ironmon_tracker/FileManager.lua

```diff
@@ -704,31 +756,34 @@ function FileManager.addCustomThemeToFile(themeName, themeCode)
 		return
 	end
 
-	local themeFilePath = FileManager.prependDir(FileManager.Files.THEME_PRESETS)
-	local file = io.open(themeFilePath, "a")
-
-	if file ~= nil then
-		file:write(string.format("%s %s", themeName, themeCode))
-		file:write("\n")
-		file:close()
-	else
-		print(string.format('> ERROR: Unable to save custom Theme "%s" to file: %s', themeName, FileManager.Files.THEME_PRESETS))
+	local folderpath = FileManager.getPathOverride("Theme Presets") or FileManager.dir
+	local filepath = folderpath .. FileManager.Files.THEME_PRESETS
+	local file = io.open(filepath, "a")
+	if not file then
+		-- Don't really want to flood the console with error messages; if important, use Main.DisplayError()
+		-- print(string.format('> ERROR: Unable to save custom Theme "%s" to file: %s', themeName, FileManager.Files.THEME_PRESETS))
+		return
 	end
```

Comment:
I agree. Definitely dont want to create multiple popups here.

I believe I have a fail-safe check elsewhere in place already, but it was never noticed because of the flood of messages from the theme error file stuff. I'm hopeful that will be visible if someone reports that annoying file/folder path issue in the future. Let's just be on the lookout for it just in case.